service_client/static_net.py

```diff
@@ -0,0 +1,324 @@
+"""Static networking related functionality."""
```

Comment:
I feel like this doesn't really belong in `service_client`.

I know the organization in this project is a bit of a mess, but the idea was for this to include just the stuff that talked to assisted service and this is more of a utility for editing and generating nmstate in isolation.